templates/example/flake.nix

```
@@ -53,7 +53,10 @@
   # see :help nixCats.flake.outputs
   outputs = { self, nixpkgs, ... }@inputs: let
     inherit (inputs.nixCats) utils;
-    luaPath = "${./.}";
+    luaPath = builtins.path {
```

```suggestion
    luaPath = self;
```

Their docs lead me to believe that for this template, this works, will have to try it out and see in a few hours.